eZ/Publish/Core/REST/Client/Values/User/User.php

```diff
@@ -35,6 +35,14 @@ public function getVersionInfo()
     }
 
     /**
+     * {@inheritdoc}
```

Comment:
I wouldn't use `{@inheritDoc}` here. First, using curly braces means that the tag is inline, while what it means is "re-use the whole parent doc". At the moment, PhpDocumentor doesn't know about the non-inline usage of that tag. There are discussions about standardizing it, but not done yet: https://phpdoc.org/docs/latest/guides/inheritance.html#the-inheritdoc-tag.

If nothing is specified, the parent's doc should be re-used. The alternative is to copy the parent's doc, like it is done for other methods in the same class.